zoo/board_games/mcts_bot.py

```diff
@@ -125,17 +159,35 @@ def n(self):
         return self._number_of_visits
```

Comment:
"""
    This property represents the number of times the node has been visited during the search.
    """